feat: adding cat painting project (#50565)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/646c48df8674cf2b91020ecc.md

---
id: 646c48df8674cf2b91020ecc
title: Step 3
challengeType: 0
dashedName: step-3
---

# --description--

> Even though you have not written any CSS yet, you can link your CSS file now.
> 
> Add a `link` element with a `rel` attribute of `stylesheet`, and an `href` of `styles.css`.

Reviewer: I prefer the wording from the build a piano project here

```suggestion
Add a `link` element within your `head` element. For that `link` element, set the `rel` attribute to `stylesheet` and the `href` to `./styles.css`.
```
